python/cdp_montana_legislature_backend/scraper.py

```diff
@@ -36,5 +40,77 @@ def get_events(
     from GitHub Actions UI.
     """
 
-    # Your implementation here
+    # TODO: Read key bills from config or Capitol Tracker JSON
+    key_bill_names = ["HB 1"]
+    
+    # Start at the big table of all bills from the 2021 session.
+    bills_url_2021 = "http://laws.leg.mt.gov/legprd/LAW0217W$BAIV.return_all_bills?P_SESS=20211"
```

Comment:
@smai-f Have you given any thought to how we might want to handle past / future legislative sessions?

Reply:
I'd say push a commit to update this line once the 2023 site is live. We should only have to update this URL once at the beginning of the 2023 session 😁  Maybe it could live in the config with the key bills

Reply:
Update: Mark raised big money, we're scraping everything!